### PR TITLE
DOC more consistent formatting in docstring

### PR DESCRIPTION
**Pull request by TheooJ**
I uniformized the keys order in `InterpolationJoiner` and `Joiner` to match the Polars and Pandas API

Small number of cosmetic changes to docstrings

### CODE REVIEW

File: skrub/_interpolation_joiner.py [hunk 1]
```diff
@@ -177,9 +177,9 @@ def __init__(
         self,
         aux_table,
         *,
+        key=None,
```
Comment (glemaitre) [inline skrub/_interpolation_joiner.py:180]: This is less of a trivial change here. I let @jeromedockes judging this one.

File: skrub/_join_utils.py [hunk 2]
```diff
@@ -40,6 +40,7 @@ def check_key(
         How to refer to `aux_key` in error messages.
     key_name : str
```
Comment (glemaitre) [inline skrub/_join_utils.py:41]: Can you add the default also for `main_key_name` and `aux_key_name`

```suggestion
    key_name : str, default="key"
```

File: skrub/_joiner.py [hunk 3]
```diff
@@ -220,19 +220,19 @@ def __init__(
         self,
         aux_table,
         *,
+        key=None,
```
Comment (glemaitre) [inline skrub/_joiner.py:223]: Also referring to @jeromedockes

**Review by glemaitre — APPROVED** [review]
Only some nitpicks

**Review by glemaitre — APPROVED** [review]
Only some nitpicks

File: skrub/_interpolation_joiner.py (hunk 1 above)
Comment (jeromedockes) [reply skrub/_interpolation_joiner.py:180]: the order proposed in this pr is the same as polars and pandas so  I think it is better

**Comment by jeromedockes** [comment]
thanks a lot @TheooJ !

**Review by jeromedockes — APPROVED** [review]

**Pull request merged**